JS/Lesson34/gulcin/recipe-app/src/modules/recipes/RecipeDetails/index.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import { Ingredients } from "./ingredients";
+import './styles.css';
+
+export const RecipeDetails = ({ id }) => {
+    const [recipe, setRecipe] = useState([]);
+
+    useEffect(() => {
+        fetch(`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${id}`)
+            .then(res => res.json())
+            .then(data => {
+                if (data && data.meals) {
+                    setRecipe(data.meals);
```

Comment:
You'd need to pick the first element of the array in the response as we only expect 1 or none recipes returned for a query by id.
```
setRecipe(data.meals[0]);
```